src/Bigcommerce/Api/Resources/Currency.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Bigcommerce\Api\Resources;
+
+use Bigcommerce\Api\Resource;
+use Bigcommerce\Api\Client;
+
+/**
+ * Represents a single currency.
+ */
+class Currency extends Resource
+{
+    protected $ignoreOnCreate = array(
+        'date_created',
+        'date_modified',
+    );
+
+    protected $ignoreOnUpdate = array(
+        'id',
+        'date_created',
+        'date_modified',
+    );
+
+    public function create()
+    {
+        return Client::createCurrency($this->getCreateFields());
+    }
+
+    public function update()
+    {
+        return Client::updateCurrency($this->id, $this->getUpdateFields());
+    }
+
+    public function delete()
+    {
+        return Client::deleteCurrency($this->id);
+    }
+}
```

Comment:
should we also support a "fetch collection" type of operation? If so we should add it here, otherwise there probably isn't a need for `Client::getCurrencies` in the API client itself